examples/cluster/multi-node/cluster.yaml

```diff
@@ -40,7 +40,6 @@ spec:
   #   * creating floating ips
   #   * creating load balancer
   externalNetworkId: <external-network-id>
-  managedSecurityGroups: false
```

Comment:
Hi, would probably need to update [config file](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/docs/config.md#security-groups) as well if we remove this.

Reply:
Why? I just removed it because it's the default value. So it shouldn't change anything

Reply:
Ohh okay, I thought maybe there is no need to mention that in config file. But looks good now